modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -684,2512 +658,2530 @@ this operation creates a zero-byte object.
-* S3:RequestTimeTooSkewed
-* S3:MissingContentLength
-* S3:MissingRequestBodyError
-* S3:UNKNOWN
-* S3:MaxMessageLengthExceeded
-* S3:InvalidURI
-* S3:PermanentRedirect
-* S3:IncompleteBody
-* S3:NoLoggingStatusForKey
-* S3:TooManyBuckets
-* S3:AccountProblem
-* S3:MissingSecurityHeader
-* S3:BucketAlreadyExists
-* S3:UserKeyMustBeSpecified
-* S3:MalformedXML
-* S3:OperationAborted
-* S3:RETRY_EXHAUSTED
-* S3:InvalidRequest
-* S3:InvalidPartOrder
-* S3:CONNECTIVITY
-* S3:MethodNotAllowed
-* S3:InvalidLocationConstraint
-* S3:InvalidTargetBucketForLogging
-* S3:RequestIsNotMultiPartContent
-* S3:NoSuchLifecycleConfiguration
-* S3:NotImplemented
-* S3:NoSuchUpload
-* S3:EntityTooLarge
-* S3:MaxPostPreDataLengthExceededError
-* S3:ServiceUnavailable
-* S3:InvalidSOAPRequest
-* S3:UnresolvableGrantByEmailAddress
-* S3:CrossLocationLoggingProhibited
-* S3:NoSuchBucketPolicy
-* S3:PreconditionFailed
-* S3:InvalidPart
-* S3:MetadataTooLarge
-* S3:MissingSecurityElement
-* S3:InternalError
-* S3:InvalidAccessKeyId
-* S3:InvalidObjectState
-* S3:InvalidBucketName
-* S3:EntityTooSmall
-* S3:InvalidToken
-* S3:InvalidStorageClass
-* S3:INVALID_RESPONSE_PRESIGNEDURL_MALFORMED
-* S3:NoSuchBucket
-* S3:InvalidAddressingHeader
-* S3:NotSignedUp
-* S3:KeyTooLong
-* S3:InvalidSecurity
-* S3:BucketNotEmpty
-* S3:RequestTimeout
-* S3:CredentialsNotSupported
-* S3:InvalidBucketState
-* S3:BucketAlreadyOwnedByYou
-* S3:InvalidEncryptionAlgorithmError
-* S3:ExpiredToken
-* S3:AmbiguousGrantByEmailAddress
-* S3:Redirect
-* S3:MissingAttachment
-* S3:TemporaryRedirect
-* S3:MalformedPOSTRequest
-* S3:InvalidPayer
-* S3:InvalidArgument
-* S3:BadDigest
-* S3:IllegalVersioningConfigurationException
-* S3:NoSuchVersion
-* S3:SlowDown
-* S3:IncorrectNumberOfFilesInPostRequest
-* S3:InvalidPolicyDocument
-* S3:InvalidRange
 
 [[getBucketCorsConfiguration]]
-== Get Bucket Cors Configuration
-
+=== Get Bucket Cors Configuration
 `<s3:get-bucket-cors-configuration>`
 
++++
 Answers the cors configuration information set for the bucket.
++++
 
+==== Parameters
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Bucket Name a| String |  The bucket whose cors to retrieve. |  | x
-| Target Variable a| String |  The name of a variable that stores the output of an operation. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the targer variable. |  `#[payload]` |
+| Bucket Name a| String |  +++The bucket whose cors to retrieve.+++ |  | x
+| Target Variable a| String |  +++The name of a variable to store the operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ |  `#[payload]` | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A r
```

Comment:
This does not appear like it's correct in the View File view. Please check this.

Reply:
This is what I see in View File view. Could you please advise me on what is incorrect here?
![image](https://user-images.githubusercontent.com/16899901/59603578-0d477c80-910a-11e9-8efe-9959688c5f03.png)

If I click on `Reconnect` or `Reconnect Forever` links it shows me description of these with correct comments.

Reply:
The issue is in the Output section for the Get Bucket Versioning Configuration.
![Screen Shot 2019-06-17 at 9 18 41 AM](https://user-images.githubusercontent.com/46829037/59607629-b62db180-90e1-11e9-8911-e54e038bea87.png)

Reply:
Thanks! It should be list of OFF, ENABLED and SUSPENDED values but I couldn't make it look like that. I updated it and now it appears syntactically correct to me but it still renders wrong. Could you please take a look?